deepstack_sdk/viz.py

```diff
@@ -51,7 +52,7 @@ def drawResponse(image,response,output_font=cv2.FONT_HERSHEY_SIMPLEX,output_font
 
                 image_arr = cv2.putText(
                     img=image_arr,
-                    text=txt + " ( " + str(100*obj.confidence)+"% )",
+                    text=(txt + (" ( " + str(100*(obj.confidence))+"% )" if show_conf else '' ))if show_label else None,
```

Comment:
For displaying object confidence, I might suggest we round the value to 2 decimal places ( long decimal values doesn't look great on drawn images ). For example, we use `str(round(obj.confidence, 2))` instead of the current `obj.confidence`.

@johnolafenwa thoughts?

Reply:
@OlafenwaMoses , rounding the confidence to 2 dp would be great for visualization. Totally support it